### ISSUE TITLE
contrary Tooltip for Full Size for Image with Resizing Plugin

### ISSUE THREAD

**Issue by MCMicS** [ckeditor/ckeditor5-image#327]
The tooltip for full Size style for images with ImageResize Plugin is contrary.
The tooltip says "Full Size image" even if the image was resized.

**Possibilities**

- Restore the original size if this setting is used (see https://github.com/ckeditor/ckeditor5-image/issues/318)
- map it to alignCenter style
- change tooltip to e.g.: Disable Text warp
This is something like disable floating within HTML or prevent text wraps around objects.

![grafik](https://user-images.githubusercontent.com/4604206/64198065-d6bb0b80-ce87-11e9-8971-7b4d84c0c93f.png)

**Comment by Mgsy** [comment]
cc @mlewand

**Comment by Reinmar** [comment]
That's a good question what we should do with this one.

In fact, I'd start with this – we should not showcase resizing with default 2 styles. We should show it with the [left, center, right styles](https://ckeditor.com/docs/ckeditor5/latest/features/image.html#configuring-image-styles). I guess it makes a lot more sense than what we have now. Would you agree @MCMicS?

As for what "Full size" should do with resized images... I have no idea. Perhaps it could reset the width. But we don't have a mechanism which would allow that now. This style just changes one class and that's it. But perhaps the current behaviour will not attract that much attention if we mention that resizing makes more sense with the "visual" set of styles.

**Comment by mlewand** [comment]
Well, I haven't think of it that way, but it's a good point that this label is misleading now.

Full size image can be for me one of two:

* 100% width of the container
* Image with its natural size

Both cases depends on #318.

Personally I'd go with the natural size.

**Comment by MCMicS** [comment]
@Reinmar: Yes I agree with it. The sample should not contains the full size style.
@mlewand I'd go with naturally size. It should then be restored to that width.

What is about to deactivate the resize of an image if full size is selected. It makes only sense if one of the other style is selected.

**Comment by Reinmar** [comment]
It's not that simple – this style means "no style". It doesn't mean "make it full size" really. It means "behave naturally" already.

But this "naturally" is overriden by resizing. The width has precedence right now and there's no way to change that because the width is an inline style. Also, changing the precedence would make it impossible to have an image take 50% but be not be an aside image.
